lib/dumped/repositories/repo_repository.rb

```diff
@@ -0,0 +1,9 @@
+class RepoRepository < Hanami::Repository
+  associations do
+    belongs_to :user
+  end
+
+  def by_language(language)
+    repos.where(language: language).to_a
```

Comment:
unfortunately you need to use `map_to(Entity)` in hanami repositories, I don't know why but in my cases automapping doesn't work :(